src/common/security/utils/ram_policy.go

```diff
@@ -0,0 +1,55 @@
+package utils
+
+import (
+	"strings"
+
+	"github.com/goharbor/harbor/src/common/ram"
+)
+
+var (
+	publicProjectPolicies = []*ram.Policy{
+		parsePolicy("image, pull"),
+	}
+)
+
+func parsePolicy(line string) *ram.Policy {
+	elements := []string{}
+
+	for _, element := range strings.SplitN(line, ",", 3) {
+		elements = append(elements, strings.TrimSpace(element))
+	}
+
+	var (
+		resource ram.Resource
+		action   ram.Action
+		effect   ram.Effect
+	)
+
+	switch len(elements) {
+	case 3:
+		resource = ram.Resource(elements[0])
+		action = ram.Action(elements[1])
+		effect = ram.Effect(elements[2])
+	case 2:
```

Comment:
need default value for effect?

Reply:
effect default value is provided in Policy struct, now parsePolicy func removed.